src/main/java/frc/utils/devicewrappers/SparkMaxConfiguration.java

```diff
@@ -0,0 +1,85 @@
+package frc.utils.devicewrappers;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import frc.utils.PIDObject;
+
+public class SparkMaxConfiguration {
+
+	public double conversionFactor;
+	public Rotation2d forwardAngleSoftLimit;
+	public Rotation2d backwardAngleSoftLimit;
+	public boolean enableForwardSoftLimit;
+	public boolean enableBackwardSoftLimit;
+	public PIDObject slot0;
+	public PIDObject slot1;
+	public PIDObject slot2;
+	public PIDObject slot3;
+	public int currentLimit;
+
+	public SparkMaxConfiguration() {
+		this.conversionFactor = 1;
+
+		this.forwardAngleSoftLimit = Rotation2d.fromDegrees(0);
+		this.backwardAngleSoftLimit = Rotation2d.fromDegrees(0);
+		this.enableForwardSoftLimit = false;
+		this.enableBackwardSoftLimit = false;
+
+		this.currentLimit = 0;
```

Comment:
nonoonon. the default current limit should never be set to 0!!!!
if you want, the default value can be 40, as the breaker we normally use there, but not 0!